candi/bashible/common-steps/node-group/031_install_containerd.sh.tpl

```diff
@@ -14,19 +14,17 @@
 
 {{- if eq .cri "Containerd" }}
 
-# install toml-merge
-bb-rp-install "toml-merge:{{ .images.registrypackages.tomlMerge01 }}"
-
 bb-event-on 'bb-package-installed' 'post-install'
 post-install() {
   systemctl daemon-reload
   systemctl enable containerd.service
+
 {{ if ne .runType "ImageBuilding" -}}
   bb-flag-set containerd-need-restart
 {{- end }}
 }
 
-if bb-yum-package? docker-ce; then
+if bb-apt-package? docker-ce || bb-apt-package? docker.io; then
```

Comment:
Need to discuss, i didn' get how it used to work in common-steps outside centos bundle and i don't get how it works now in centos node.

Reply:
good point, i'll restore original scripts